src/utils/jws.ts

```diff
@@ -0,0 +1,65 @@
+import isPlainObject from 'lodash/isPlainObject.js';
+
+import { Encoder } from './encoder.js';
+import { GeneralJws } from '../jose/jws/general/types.js';
+import { PublicJwk } from '../jose/types.js';
+import { SignatureEntry } from '../jose/jws/general/types.js';
+import { signers as verifiers } from '../jose/algorithms/signing/signers.js';
+
+/**
+ * Utility class for JWS related operations.
+ */
+export class Jws {
```

Comment:
Curious why no tests for this class?

Reply:
No no, good question. Mainly because:

1. There are a couple tests declared (not implemented) in `verifier.spec.ts`, I have now moved those tests to `jws.spec.ts`.
2. The code is indirectly tested 100s of times in other tests, thus these direct unit tests never got treated high pri.
3. The intent of this PR was minor refactoring, not changing/adding tests.